### DISCUSSION TITLE
Cannot run my Script ....Can someone assist

### DISCUSSION THREAD

**Discussion by sgmffleong0297** [getgauge/taiko#1968] [General]
I am a beginner to programming  , type below , the script cannot execute. FYI, I have only installed Taiko but not JavaScript program on my PC. Do I have to install JavaScript program to execute it.

C:\Users\user>npx taiko OCCTK1.js --observe
File does not exist.

**Comment by sriv** [comment]
hi, yes you need javascript installed. Specifically nodejs - https://nodejs.org/en/download/

`npx` is a command that would come with nodejs.

**Reply by sgmffleong0297** [reply]
have already installed NodeJs.  Cannot run my script:

> npx taiko OCCTK1.js --observe
npx taiko OCCTK1.js --observe
    ^^^^^

Uncaught SyntaxError: Unexpected identifier

**Reply by sriv** [reply]
What do `npx --version` and `node --version` commands give as output? If you have node installed and available in `PATH` it should print out the version numbers.

And once this is sorted, you should be able to.ru no the `npx taiko` command

**Reply by sgmffleong0297** [reply]
Still cannot run:

C:\Users\user>node --version
v14.16.0

C:\Users\user>npx --version
6.14.11

C:\Users\user>npx taiko

Version: 1.2.3 (Chromium: 88.0.4314.0)
Type .api for help and .exit to quit

> npx taiko OCCTK3test.js--observe
Uncaught Missing semicolon (1:18)
> Uncaught Unexpected identifier

**Reply by sriv** [reply]
So this seems like a different error, and it seems to suggest that `OzcCTzK3test.js` file has a syntax error.

It's tough to figure out the actual cause without looking at the file.

**Reply by sgmffleong0297** [reply]
const { openBrowser, goto, text, below, click, textBox, into, write, closeBrowser } = require('taiko');
(async () => {
    try {
        await openBrowser();
        await goto("https://www.orchidclub.com");
        await click("weekday 18-hole & 9-Hole",(below(text("online booking of tee times"))));
        await write("sgmik@gmail.com",into(textBox(below(text("Email address")))));
        await write("90737617",into(textBox(below(text("Mobile Number")))));
        await write("15008380",into(textBox(below(text("Membership Number of Player 1")))));
        await write("Tan Kim Lian",into(textBox(below(text("Full Name of Player 1")))));
        await click("18 hole");
        await write("0735AM",into(textBox(below(text("Preferred tee time")))));
        await write("15008551",into(textBox(below(text("Membership Number of Player 2")))));
        await write("Lee Kok Pheng",into(textBox(below(text("Full Name of Player 2")))));
        await write("MGK 007532",into(textBox(below(text("Membership Number of Player 3")))));
        await write("Thomas Low",into(textBox(below(text("Full Name of Player 3")))));
        await write("04",into(textBox(below(text("MM")))));
        await write("06",into(textBox(below(text("DD")))));
        await write("2020",into(textBox(below(text("YYYY")))));
        
    } catch (error) {
        console.error(error);
    } finally {
        await closeBrowser();
    }
})();
